src/main/java/com/comphenix/protocol/wrappers/WrappedLevelChunkData.java

```diff
@@ -163,11 +166,19 @@ public static class LightData extends AbstractWrapper {
                     MinecraftReflection.getPacketDataSerializerClass(), int.class, int.class);
             BIT_SET_ACCESSORS = Accessors.getFieldAccessorArray(HANDLE_TYPE, BitSet.class, true);
             BYTE_ARRAY_LIST_ACCESSORS = Accessors.getFieldAccessorArray(HANDLE_TYPE, List.class, true);
-            TRUST_EDGES_ACCESSOR = Accessors.getFieldAccessor(reflection.getField(FuzzyFieldContract.newBuilder()
-                    .typeExact(boolean.class)
-                    .build()));
+            System.out.println(Bukkit.getVersion());
```

Comment:
Can we remove those debugs?

Reply:
Yes, thank you for pointing it out. I removed them